src/app/components/BibPage/Tabbed.jsx

```diff
@@ -74,7 +74,7 @@ class Tabbed extends React.Component {
           { this.props.tabs.map((tab, i) => {
             let j = i + 1;
             return (
-              <li id={`tab${j}`} className={(parseInt(this.state.tabNumber) === j ? 'activeTab' : null) } role='presentation'>
+              <li id={`tab${j}`} key={`tab${j}`} className={(parseInt(this.state.tabNumber) === j ? 'activeTab' : null) } role='presentation'>
```

Comment:
I don't think it is big issue, but here and line 99 the key will start with 1 but not 0 while other places the key will start with 0.